### PR TITLE
CRAFTERCMS-2415;CRAFTERCMS-2416

### PR DESCRIPTION
**Pull request by dejan-brkic**
Auth mechanics updated for LDAP and Headers to match new group structure

### Ticket reference or full description of what's in the PR
https://github.com/craftercms/craftercms/issues/2415
https://github.com/craftercms/craftercms/issues/2416

### CODE REVIEW

File: src/main/java/org/craftercms/studio/api/v1/constant/StudioConstants.java [hunk 1]
```diff
@@ -119,6 +119,7 @@
      * System constants
      */
     String FILE_SEPARATOR = "/";
+    String ORGANIZATION_ADMIN_GROUP = "org_admin";
```
Comment (sumerjabri) [inline src/main/java/org/craftercms/studio/api/v1/constant/StudioConstants.java:122]: Should this become `system_admin` per new naming of super admin?

File: src/main/java/org/craftercms/studio/impl/v1/service/security/SecurityServiceImpl.java [hunk 2]
```diff
@@ -851,8 +851,7 @@ private boolean isAdmin(String username) {
         boolean toRet = false;
         if (CollectionUtils.isNotEmpty(userGroups)) {
             for (Group group : userGroups) {
-                if (StringUtils.equalsIgnoreCase(group.getName(),
-                        studioConfiguration.getProperty(SECURITY_GLOBAL_ADMIN_GROUP))) {
+                if (StringUtils.equalsIgnoreCase(group.getName(), ORGANIZATION_ADMIN_GROUP)) {
```
Comment (sumerjabri) [inline src/main/java/org/craftercms/studio/impl/v1/service/security/SecurityServiceImpl.java:854]: Update this one too to system admin

File: src/main/java/org/craftercms/studio/impl/v1/web/security/access/StudioAbstractAccessDecisionVoter.java [hunk 3]
```diff
@@ -129,8 +129,7 @@ protected boolean isAdmin(UserDAO user) {
         boolean toRet = false;
         if (CollectionUtils.isNotEmpty(userGroups)) {
             for (Group group : userGroups) {
-                if (StringUtils.equalsIgnoreCase(group.getName(),
-                        studioConfiguration.getProperty(SECURITY_GLOBAL_ADMIN_GROUP))) {
+                if (StringUtils.equalsIgnoreCase(group.getName(), ORGANIZATION_ADMIN_GROUP)) {
                     toRet = true;
```
Comment (sumerjabri) [inline src/main/java/org/craftercms/studio/impl/v1/web/security/access/StudioAbstractAccessDecisionVoter.java:133]: Sys admin again

**Review by sumerjabri — CHANGES_REQUESTED** [review]

File: src/main/java/org/craftercms/studio/impl/v1/web/security/access/StudioAbstractAccessDecisionVoter.java (hunk 3 above)
Comment (dejan-brkic) [reply src/main/java/org/craftercms/studio/impl/v1/web/security/access/StudioAbstractAccessDecisionVoter.java:133]: Yes, but I have SQL scripts in different PR. I didn't want to cause merge conflicts. Will update constants accordingly once everything is merged nicely without conflicts

File: src/main/java/org/craftercms/studio/impl/v1/service/security/SecurityServiceImpl.java (hunk 2 above)
Comment (dejan-brkic) [reply src/main/java/org/craftercms/studio/impl/v1/service/security/SecurityServiceImpl.java:854]: Yes, but I have SQL scripts in different PR. I didn't want to cause merge conflicts. Will update constants accordingly once everything is merged nicely without conflicts

File: src/main/java/org/craftercms/studio/api/v1/constant/StudioConstants.java (hunk 1 above)
Comment (dejan-brkic) [reply src/main/java/org/craftercms/studio/api/v1/constant/StudioConstants.java:122]: Yes, but I have SQL scripts in different PR. I didn't want to cause merge conflicts. Will update constants accordingly once everything is merged nicely without conflicts

**Pull request merged**